notebooks/monte_carlo_case_study_nigeria_data.ipynb

```diff
@@ -0,0 +1,549 @@
+    "import os\n",
+    "import sys\n",
+    "\n",
+    "if not os.path.isdir(\"pypsa-earth\"):\n",
+    "    os.chdir(\"../..\")\n",
+    "sys.path.append(os.getcwd()+\"/pypsa-earth/scripts\")\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/gbotemi/miniconda3/envs/pypsa-earth/lib/python3.10/site-packages/pypsa/networkclustering.py:16: UserWarning: The namespace `pypsa.networkclustering` is deprecated and will be removed in PyPSA v0.24. Please use `pypsa.clustering.spatial instead`. \n",
+      "  warnings.warn(\n"
+     ]
+    }
+   ],
+   "source": [
+    "import os\n",
+    "import pypsa\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
+    "from IPython.display import Image\n",
+    "import statsmodels.api as sm\n",
+    "from scipy.stats.mstats import zscore\n",
+    "from scipy import stats\n",
+    "from scripts.monte_carlo import (\n",
+    "            monte_carlo_sampling_pydoe2, monte_carlo_sampling_scipy, \n",
+    "            monte_carlo_sampling_chaospy, validate_parameters)\n",
+    "import logging\n",
+    "import yaml\n",
+    "import warnings\n",
+    "warnings.filterwarnings('ignore')\n",
+    "\n",
+    "pd.set_option('display.max_columns', 500)\n",
+    "\n",
+    "# disable logging from imported modules \n",
+    "logging.disable(sys.maxsize)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# load the config file used for the monte carlo simulation\n",
+    "with open(\"mc/pypsa-earth/config.yaml\", \"r\") as stream:\n",
+    "    config = yaml.safe_load(stream)[\"monte_carlo\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Loading simulation results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/Users/gbotemi/Documents/code/PYPSA\n"
+     ]
+    }
+   ],
+   "source": [
+    "!pwd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "All Networks loaded successfully\n"
+     ]
+    }
+   ],
+   "source": [
+    "# load solved networks into a list\n",
+    "network_list = []\n",
+    "\n",
+    "for count in range(0,49):\n",
+    "    # set directory of solved networks in results folder\n",
+    "    path = f\"mc/pypsa-earth/results/NG/networks/elec_s_10_ec_lcopt_Co2L-24H_m{count}.nc\"\n",
+    "    network = pypsa.Network(path)\n",
+    "    network_list.append(network)\n",
+    "\n",
+    "print(\"All Networks loaded successfully\")"
```

Comment:
Doesn't it make sense to use logging instead of prints? We have anyway loaded `logging` import.

Reply:
Yh, Totally fine to use logging.